### PR TITLE
[BULK UPDATE] DocuTune - Fix build validation issues: docs-link-absolute (part 1)

### PR DESCRIPTION
**Pull request by alexbuckgit**
**Global effort to fix build validation errors**

The Microsoft Skilling team is fixing build validation errors and specific brand inconsistencies in Microsoft technical documentation for the rest of FY23 Q1. This will help address various accessibility, security, and usability issues. This PR was generated using DocuTune. It includes only targeted fixes and minor formatting adjustments.

DocuTune PRs improve quality of technical content by finding and automatically correcting a broad set of issues only where the correction is suitable. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

CorrelationId: 0df9c2d6-63f8-4d1e-8b57-b33611a06514
InitiativeId: docutune-build-validation-issues-2022-05
PointOfContact: Alex Buck

### CODE REVIEW

**Comment by alexbuckgit** [comment]
#docutune-draft

File: mobile-blazor-bindings/advanced/custom-components.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ You can write reusable custom components either using Blazor components or by wr
 
 You can create a reusable Blazor component by writing a `.razor` file just like you would in an application. The component can be shipped as a NuGet package so that it is easily reusable in other projects.
 
-To learn more about component libraries, check out the [ASP.NET Core Razor components class libraries](https://docs.microsoft.com/aspnet/core/blazor/class-libraries&tabs=visual-studio).
+To learn more about component libraries, check out the [ASP.NET Core Razor components class libraries](/aspnet/core/blazor/class-libraries&tabs=visual-studio).
```
Comment (JustPies) [inline mobile-blazor-bindings/advanced/custom-components.md:17]: This link is broken.

**Comment by JustPies** [comment]
#docutune-wait

**Comment by alexbuckgit** [comment]
@lachlanwgordon

Please merge the changes in this PR. For more information, see the description at the top of this PR.

Any pre-existing build validation issues or insufficient Acrolinx scores should be addressed in a separate PR.

#docutune-request-approval

**Pull request merged**